pkg/webui/console/containers/application-title-section/index.js

```diff
@@ -52,9 +52,9 @@ import { selectApiKeysTotalCount } from '@console/store/selectors/api-keys'
 
 const m = defineMessages({
   lastSeenAvailableTooltip:
-    'The elapsed time since the network registered activity (sent uplinks, confirmed downlinks or (re)join requests) of the end device(s) in this application.',
+    'The elapsed time since the network registered activity (sent uplinks, confirmed downlinks or (re)join requests) of the end device(s) in this application.(old)',
```

Comment:
What's the reason for this change? If these messages are outdated shouldn't we update them or remove?

Reply:
I guess this is a workaround to avoid having duplicate message definitions since this whole component will be deleted anyway.

Reply:
Yes correct. Do we delete the components that will not be used anymore right away? @kschiffer @ryaplots

Reply:
Yeah, I think we should.